packages/cli/test/test-utils.js

```diff
@@ -49,96 +49,65 @@ exports.executeGenerator = function(GeneratorOrNamespace, settings) {
  */
-exports.givenAnApplicationDir = function(tmpDir, options) {
+exports.createLB4Project = function(rootDir, options) {
   options = options || {};
-  const srcDir = path.join(tmpDir, 'src');
-  fs.writeFileSync(
-    path.join(tmpDir, 'package.json'),
-    JSON.stringify({
-      keywords: ['loopback'],
-    }),
-  );
-  fs.mkdirSync(srcDir);
-  if (!options.omitModelDir) {
-    fs.mkdirSync(path.join(srcDir, 'models'));
+  const context = {};
+  const content = {};
+  if (!options.excludeKeyword) {
+    content.keywords = ['loopback'];
   }
-  if (!options.omitRepositoryDir) {
-    fs.mkdirSync(path.join(srcDir, 'repositories'));
+
+  if (!options.excludePackageJSON) {
+    fs.writeFileSync(
+      path.join(rootDir, 'package.json'),
+      JSON.stringify(content),
+    );
   }
-  if (!options.omitControllerDir) {
-    fs.mkdirSync(path.join(srcDir, 'controllers'));
+
+  if (!options.excludeYoRcJSON) {
+    fs.writeFileSync(path.join(rootDir, '.yo-rc.json'), JSON.stringify({}));
   }
-};
 
-/**
- * Return the default path for the specified repository and temp directory.
- * @param {string=} tmpDir The temporary directory path. If omitted, the
- * returned path will be relative (prefixed with either "/" or "\",
- * @param {string} fileName The repository name.
- */
-exports.givenARepositoryPath = function(tmpDir, fileName) {
-  return exports.givenAnArtifactPath(tmpDir, 'repositories', fileName);
-};
+  fs.mkdirSync(path.join(rootDir, 'src'));
 
-/**
- * Return the default path for the specified model and temp directory.
- * @param {string=} tmpDir The temporary directory path. If omitted, the
- * returned path will be relative (prefixed with either "/" or "\",
- * depending on OS).
- * @param {string} fileName The model name.
- */
-exports.givenAModelPath = function(tmpDir, fileName) {
-  return exports.givenAnArtifactPath(tmpDir, 'models', fileName);
-};
+  if (!options.excludeControllersDir) {
+    fs.mkdirSync(path.join(rootDir, 'src', 'controllers'));
+  }
 
-/**
- * Return the default path for the specified controller and temp directory.
- * @param {string=} tmpDir The temporary directory path. If omitted, the
- * returned path will be relative (prefixed with either "/" or "\",
- * depending on OS).
- * @param {string} fileName The controller name.
- */
-exports.givenAControllerPath = function(tmpDir, fileName) {
-  return exports.givenAnArtifactPath(tmpDir, 'controllers', fileName);
-};
+  if (!options.excludeModelsDir) {
+    fs.mkdirSync(path.join(rootDir, 'src', 'models'));
+  }
 
-/**
- * @param {string=} tmpDir The temporary directory path. If omitted, the
- * returned path will be relative (prefixed with either "/" or "\",
- * depending on OS).
- * @param {string} artifactDir The artifact directory name.
- * @param {string} fileName The artifact fileName.
- */
-exports.givenAnArtifactPath = function(tmpDir, artifactDir, fileName) {
```

Comment:
I think we may need to reintroduce fine-grained helpers like this one in the future, where the options provided by the single helper `createLB4Project` become not flexible enough. (For example: what if we want to create LB4 project, run a generator, then add some artifacts and then run another generator?)

Having said that and considering [YAGNI](https://www.martinfowler.com/bliki/Yagni.html), I am fine with removing these helpers since they are not used by any tests after your changes.